integration_tests/pages/assess/matchingInformationPage.ts

```diff
@@ -7,6 +7,11 @@ export default class MatchingInformationPage extends AssessPage {
     super('Matching information', assessment, 'matching-information', 'matching-information', '')
   }
 
+  shouldNotShowMentalHealthAps() {
+    cy.get(`input[name="apType"][value="mhapElliottHouse"]`).should('not.exist')
+    cy.get(`input[name="apType"][value="mhapStJosephs"]`).should('not.exist')
```

Comment:
Should this be?

```suggestion
    cy.get(`input[name="apType"][value="isMHAPElliottHouse"]`).should('not.exist')
    cy.get(`input[name="apType"][value="isMHAPStJosephs"]`).should('not.exist')
```

I think `placementCriteriaUtils` might need updating for this?

Reply:
Good spot with the integration tests.

 I believe `placementCriteriaUtils` is driven by the same apTypes we use in the `apType` page

Reply:
Ah so it is!

Reply:
I changed it whilst you were on leave  👍